src/Util/Temporal.php

```diff
@@ -122,13 +122,12 @@ public static function getTimezoneField($name = 'timezone', $label = '', $curren
 	 * @brief Wrapper for date selector, tailored for use in birthday fields.
 	 *
 	 * @param string $dob Date of Birth
+	 * @param string $timezone
 	 * @return string Formatted HTML
-	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
+	 * @throws \Exception
 	 */
-	public static function getDateofBirthField($dob)
+	public static function getDateofBirthField($dob, $timezone = 'UTC')
```

Comment:
Typehint `string` and `string` :)

Reply:
Isn't it useless since it's the lowest constraint type? Like everything is string by default?

Reply:
What about `False` is given as argument? Would it create an exception of would it be casted to an empty string? Or would it be passed as boolean? I've the fear that the last thing would occur and than at another point of the code, a boolean type isn't valid and create an exception at other places.

Reply:
Fair enough, I thought it was the lowest common parameter type denominator and as such could be skipped.